1/styles/scss/main.scss

```diff
@@ -0,0 +1,115 @@
+@import "variables";
+
+@import "base";
+
+@import "layout";
+
+@import "state";
+
+//------------HEADER---------------
+
+.header-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  justify-content: center;
+  z-index: 10;
+  width: 100vw;
+
+  li{
+    position: relative;
+  }
+
+  nav li a, .btn {
+    color: white;
+    text-transform: uppercase;
+    font-size: 12px;
+  }
+
+  li:hover{
+    &::after{
+      content: '';
+      width: 20px;
+      height: 1.5px;
+      background: white;
+      position: absolute;
+      bottom: 0;
+      left: 30%;
+    }
+  }
+
+  .btn:hover{
+    background: darken($accent, 10%) ;
+    border-color: transparent;
+    box-shadow: 0 0 10px $accent;
+  }
+}
+
+.logo {
+  padding: 20px 80px;
+}
+
+.btn {
+  display: block;
+  padding: 10px 30px;
+  border: 1px solid white;
+  border-radius: 20px;
+  background: transparent;
+}
+//-------------HEADING \ MAIN------------------
```

Comment:
header is still not centered 
![Uploading image.png…]()

Reply:
done